source/aggregation-tutorials/unpack-arrays.txt

```diff
@@ -0,0 +1,207 @@
+.. _node-aggregation-arrays:
+
+=======================
+Unpack Arrays and Group
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: code example, node.js, analyze, array
+
+Introduction
+------------
+
+In this tutorial, you can learn how to use the {+driver-short+} to
+construct an aggregation pipeline, perform the
+aggregation on a collection, and print the results by completing and
+running a sample app. This aggregation performs the following operations:
+
+- Unwind an array field into separate documents
+- Matches a subset of documents by a field value
+- Groups documents by common field values
+- Adds computed fields to each result document
```

Comment:
Issue: Make these parallel

```suggestion
- Unwind an array field into separate documents
- Match a subset of documents by a field value
- Group documents by common field values
- Add computed fields to each result document
```

Reply:
Suggestion: If there is reference documentation for these operations, link those pages here. I didn't see anything specific to this driver, but maybe in the server manual?

Reply:
The specific aggregation operators are linked in the tutorial steps, this section in the introduction is just a high level plan language explanation of the tutorial!